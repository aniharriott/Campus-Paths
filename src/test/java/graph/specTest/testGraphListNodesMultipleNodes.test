# Test Graph listNodes multiple nodes file.
# Tests that multiple nodes are printed
# when there are multiple nodes in the graph.

# Create graph
CreateGraph graph1

# Add nodes
AddNode graph1 n1
AddNode graph1 n2
AddNode graph1 n3

# Print nodes in graph
ListNodes graph1