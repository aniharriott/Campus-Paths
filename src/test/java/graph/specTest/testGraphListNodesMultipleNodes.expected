# Test Graph listNodes multiple nodes file.
# Tests that multiple nodes are printed
# when there are multiple nodes in the graph.

# Create graph
created graph graph1

# Add nodes
added node n1 to graph1
added node n2 to graph1
added node n3 to graph1

# Print nodes in graph
graph1 contains: n1 n2 n3